spec/compiler/lexer/lexer_spec.cr

```diff
@@ -230,6 +230,7 @@ describe "Lexer" do
   it_lexes_number :i8, ["0i8", "0"]
 
   it_lexes_char "'a'", 'a'
+  it_lexes_char "'\\a'", '\a'
```

Comment:
I think to keep release compatibility you would have to write `7.chr` not `'\a'`